pkg/state/steps_uki.go

```diff
@@ -611,3 +615,100 @@ func (s *State) ExtractCerts(g *herd.Graph, opts ...herd.OpOption) error {
 		return nil
 	}))...)
 }
+
+// TransitionSysext is a workaround for upgrades from 3.3.x to 3.4.x
```

Comment:
Transition -> Migrate ? (in more places in this diff)

Transition usually means change between 2 different states. I'm not a native English speaker but `migrate` sound more correct to me.